package.json

```diff
@@ -286,7 +286,7 @@
     "@metamask/post-message-stream": "^8.0.0",
     "@metamask/ppom-validator": "^0.27.0",
     "@metamask/providers": "^14.0.2",
-    "@metamask/queued-request-controller": "^0.3.0",
+    "@metamask/queued-request-controller": "^0.6.0",
```

Comment:
This is resulting in a bunch of peer dependency warnings

TODO: Validate that they're safe to ignore, or bump the other packages as well in the same PR

Reply:
Seems like theres only the one warning about queued request controller using a different version of selected network controller:

```
@metamask/selected-network-controller is listed by your project with version 9.0.0, which doesn't satisfy what @metamask/queued-request-controller (p81a9c) requests (^10.0.0).
```

the main change in 10.x.x is the update to base controller. The API afaik is the same, the typings have changed a bit here and there, but we arent getting any type errors so I think we are good to go.

https://github.com/MetaMask/core/blob/main/packages/selected-network-controller/CHANGELOG.md#1000